mozilla-mobile/android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -125,6 +125,15 @@
     <!-- Pressing this will dismiss the dialog and not refresh the webpage -->
     <string name="mozac_feature_prompt_repost_negative_button_text">Cancel</string>
 
+    <!-- Strings show in a dialog that appear when user confirm folder upload -->
+    <string name="mozac_feature_prompt_folder_upload_confirm_title">Upload files?</string>
+    <!-- Prompt message displayed when user selects a specific folder. %1%s is a folder / directory name. -->
+    <string name="mozac_feature_prompt_folder_upload_confirm_message">Make sure you trust this site before you upload from “%1$s”.</string>
```

Comment:
Raised the issue with the comment and text (from vs to) in https://phabricator.services.mozilla.com/D245522 and https://mozilla-hub.atlassian.net/browse/UXREQ-385